enginetest/queries/queries.go

```diff
@@ -9285,12 +9285,6 @@ from typestable`,
 			{"1.5000"},
 		},
 	},
-	{
```

Comment:
Where did this test go?

Reply:
`server_uuid` was getting initialized to `uuid.New()` each time the server started. This ensured there was always a server UUID, but it wasn't useful, since it changed every time the server restarted. As a binlog replica, it didn't make a difference, but as a binlog primary, it does since the replica uses the primary server's UUID as part of the GTID set identifier. 

`server_uuid` now defaults to `nil` and users will need to set or persist it before they can use replication. I think that's okay for now, but I'd like to move closer to MySQL's behavior where `server_uuid` is initialized the first time a server is started and automatically persisted. I think we shouldn't persist it in the standard `~/.dolt` data director for persisted system vars though, since then it's the `server_uuid` for any server that host starts. This is on my list to come back to, but I think simply defaulting to `nil` for now and having users set it explicitly is alright in the meantime.